### DISCUSSION TITLE
contacts plugin not installing

### DISCUSSION THREAD

**Discussion by TechDhanhind** [ionic-team/capacitor#5315] [Q&A]
When i try to install "npm install @awesome-cordova-plugins/contacts" in ionic 6, am getting error as follows

npm ERR! 404 Not Found - GET https://registry.npmjs.org/@awesome-cordova-plugins%2fcontacts - Not found
npm ERR! 404
npm ERR! 404  '@awesome-cordova-plugins/contacts@*' is not in this registry.
npm ERR! 404 You should bug the author to publish it (or use the name yourself!)
npm ERR! 404
npm ERR! 404 Note that you can also install from a
npm ERR! 404 tarball, folder, http url, or git url. ```

**Comment by jcesarmobile** [comment]
Contacts plugin was marked as deprecated back in 2018
https://github.com/danielsogl/awesome-cordova-plugins/commit/c2fe5f83366a17738d6f6713eee19729f2660001

And removed back in September when the project was still called ionic-native, so there is no contacts plugin in awesome-cordova-plugins
https://github.com/danielsogl/awesome-cordova-plugins/commit/5f891f636678fbaebc82e47f7364b877a33b5c51

If you really need it, you'll have to use latest version of [@ionic-native/contacts](https://www.npmjs.com/package/@ionic-native/contacts), but not sure if you can have both ionic-native and awesome-cordova-plugins installed or if that would cause issues.

You can also try [@capacitor-community/contacts](https://www.npmjs.com/package/@capacitor-community/contacts) plugin instead of the cordova one.
